language/rag_evaluation/deepeval_with_gemini.ipynb

```diff
@@ -0,0 +1,1170 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "executionInfo": {
+     "elapsed": 3,
+     "status": "ok",
+     "timestamp": 1708026667086,
+     "user": {
+      "displayName": "",
+      "userId": ""
+     },
+     "user_tz": 300
+    },
+    "id": "ijGzTHJJUCPY",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Copyright 2023 Google LLC\n",
+    "#\n",
+    "# Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "#     https://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "VEqbX8OhE8y9"
+   },
+   "source": [
+    "# Getting Started with the Vertex AI Gemini API & Python SDK\n",
```

Comment:
Change this title to be more specific to this notebook.

Reply:
Done